x-pack/plugins/alerting/server/application/rule/methods/bulk_edit/bulk_edit_rules.ts

```diff
@@ -308,9 +321,42 @@ async function bulkEditRulesOcc<Params extends RuleParams>(
   }
   await rulesFinder.close();
 
+  const updatedInterval = rules
+    .filter((rule) => rule.attributes.enabled)
+    .map((rule) => rule.attributes.schedule?.interval)
+    .filter(isValidInterval);
+
+  try {
+    await validateScheduleLimit({
```

Comment:
we can do this check only when we tried to update the interval of rule. It will be better for the speed of the other tasks

Reply:
https://github.com/elastic/kibana/pull/164791/commits/74a53604d4cad622dc43ad41d53d08f8cddcb859